authorization_services/topics/resource-server-create-client.adoc

```diff
@@ -10,12 +10,22 @@ The first step to enable {project_name} Authorization Services is to create the
 .Clients
 image:{project_images}/resource-server/client-list.png[alt="Clients"]
 
+ifeval::[{project_community}==true]
+. On this page, click *Create client*.
+endif::[]
+ifeval::[{project_product}==true]
 . On this page, click *Create*.
+endif::[]
 +
 .Add Client
 image:{project_images}/resource-server/client-create.png[alt="Add Client"]
 
 . Type the `Client ID` of the client. For example, _my-resource-server_.
+ifeval::[{project_community}==true]
+. Click *Next*.
+. Toggle *Client authentication* to ON.
+. Click *Save*.
```

Comment:
```suggestion
. Click *Save*.
. Scroll down to the *Capability config* section.
```

Reply:
Line 39 should be:
ifeval::[{project_product}==true]
. Click *Save*. 
+
The client is created and the client Settings page opens. A page similar to the following is displayed:
endif::[]
ifeval::[{project_community}==true]
. Click *Save*. 
+
The client is created.
endif::[]